src/components/studies/launch/IrbDetails.tsx

```diff
@@ -192,10 +186,27 @@ const IrbDetails: React.FunctionComponent<IrbDetailsProps> = ({
     return newContactsArray
   }
 
+  const getDateWithTimeZone = (date: Date) => {
+    // code from: https://stackoverflow.com/questions/7556591/is-the-javascript-date-object-always-one-day-off
+    return new Date(date.getTime() - date.getTimezoneOffset() * -60000)
+  }
+
+  const getFormattedDate = (date: Date | null) => {
+    return date ? moment(date).format('YYYY-MM-DD') : ''
+  }
+
   const displayApprovalDateError =
-    studyData?.irbApprovalDate &&
-    studyData?.irbApprovedUntil &&
-    studyData?.irbApprovalDate > studyData?.irbApprovedUntil
+    study.irbDecisionOn &&
+    study.irbExpiresOn &&
+    study.irbDecisionOn > study.irbExpiresOn
+  const irbDecisionIsApproved =
+    !study.irbDecisionType || study.irbDecisionType === 'approved'
+  const irbDecisionDate = study.irbDecisionOn
+    ? getDateWithTimeZone(new Date(study.irbDecisionOn))
```

Comment:
I think toLocaleDateString() or one of the moment functions should do the trick. I  don't think we need to the custom function here

Reply:
OK - we might revisit it later -- but I can see the complexity. looks good